src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -672,6 +814,10 @@ def launch_fluent(
         which means an independent watchdog process is run to ensure
         that any local GUI-less Fluent sessions started by PyFluent are properly closed (or killed if frozen)
         when the current Python process ends.
+    slurm : bool, optional
```

Comment:
can make it as `slurm_dict` with other slurm options similar to `container_dict`